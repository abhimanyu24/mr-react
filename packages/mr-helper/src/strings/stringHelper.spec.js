import {pluralize, underscore, } from "./stringHelper";

describe("stringHelper.js", () => {
  it("pluralizes string", () => {
    expect(
      pluralize("mouse")
    ).toEqual("mice");
    expect(
      pluralize("mice", true)
    ).toEqual("mouse");
    
    expect(
      pluralize("userResponse")
    ).toEqual("userResponses");

    // expect(
    //   validator.isValid("key", "hellogmail.com", "required|email")
    // ).toEqual(false);
  });
  it('underscores string', () => {
    // expect(app).toMatchSnapshot();
  });
  
});