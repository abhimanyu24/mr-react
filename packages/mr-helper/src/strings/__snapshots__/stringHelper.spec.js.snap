// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stringHelper.js underscores string: user_response 1`] = `"user_response"`;
